### PR TITLE
add section access/map rules

### LINKED ISSUE #4: Create and open section
NtCreateSection, ZwCreateSection, NtOpenSection, ZwOpenSection
- https://ired.team/offensive-security/code-injection-process-injection/ntcreatesection-+-ntmapviewofsection-code-injection
- https://www.cyberbit.com/blog/endpoint-security/malware-mitigation-when-direct-system-calls-are-used/

### PR DESCRIPTION
**Pull request by williballenthin**
closes #4

### CODE REVIEW

File: host-interaction/process/inject/inject-apc.yml [hunk 1]
```diff
@@ -13,9 +13,7 @@ rule:
     - and:
       - or:
         - match: write process memory
-        - api: kernel32.MapViewOfSection
```
Comment (williballenthin) [inline host-interaction/process/inject/inject-apc.yml:16]: `kernel32.MapViewOfSection` is not an API exported by kernel32, oops

File: host-interaction/process/access-section-object.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: access section object
```
Comment (mr-tz) [inline host-interaction/process/access-section-object.yml:3]: library rule or worth to report on its own?

Comment (mr-tz) [inline host-interaction/process/access-section-object.yml:3]:
```suggestion
    name: create or open section object
```

Comment (mr-tz) [reply host-interaction/process/access-section-object.yml:3]: to stay consistent with file/registry rules?

File: host-interaction/process/inject/inject-apc.yml [hunk 3]
```diff
@@ -13,9 +13,7 @@ rule:
...
-        - api: NtMapViewOfSection
-        - api: ZwMapViewOfSection
+        - match: map section object
```
Comment (williballenthin) [inline host-interaction/process/inject/inject-apc.yml:16]:
```suggestion
        - match: create or open section object
```

File: lib/create-or-open-section-object.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: create or open section object
+    namespace: host-interaction/process
```
Comment (mr-tz) [inline lib/create-or-open-section-object.yml:4]:
```suggestion
```

**Review by mr-tz — APPROVED** [review]

**Pull request merged**